### ISSUE TITLE
Multiple conversation branches.

### ISSUE THREAD

**Issue by lennartle** [cogentapps/chat-with-gpt#111]
The most important feature of the original ChatGPT web interface is that you can follow the conversation from any point and preserve the previous dialogue branch. For now, if you try to edit a message, the whole response branch will be deleted. Uncool, bruh.

**Comment by cogentapps** [comment]
Agreed, today's update lays some of the groundwork for this feature, so hopefully it can be added soon.
